src/Sarif.Viewer.VisualStudio/ErrorList/SarifErrorListEventProcessor.cs

```diff
@@ -22,6 +22,10 @@ internal class SarifErrorListEventProcessor : TableControlEventProcessorBase, IS
         private SarifErrorListItem currentlySelectedItem;
         private SarifErrorListItem currentlyNavigatedItem;
 
+        internal SarifErrorListEventProcessor()
```

Comment:
>SarifErrorListEventProcessor [](start = 17, length = 28)

we could remove this, since we are not using it.

Reply:
done